src/FE/FEsrc/components/MenuHeader.tsx

```diff
@@ -148,36 +235,87 @@ const MenuHeader: React.FC = () => {
         <Header>
             <HeaderComponents>
                 <Link to='/'>
-                    <img
-                        src={require('../images/Logo.svg')}
-                        height={100}
-                        style={{ paddingRight: 20, cursor: 'pointer' }}
-                    />
+                    {isSmallScreen ? (
+                        <img
+                            src={require('../images/MiniLogo.svg')}
+                            height={100}
```

Comment:
changing height to 90 vertically centers the logo a bit better, but not too important

Reply:
updated